adafruit_atecc/adafruit_atecc.py

```diff
@@ -488,7 +535,14 @@ def _write(self, zone, address, buffer):
         self._get_response(status)
         self.idle()
 
-    def _read(self, zone, address, buffer):
+    def _read(self, zone: Any, address: int, buffer: bytearray):
```

Comment:
inside this function `zone` gets passed to `_send_command()` as the 2nd argument. In the annotations for `_send_command()` you've got all of the first 3 arguments typed as `int`. 

If those types in the `_send_command()` arguments are correct I think it means that `zone` here should be an `int` instead of `Any`